dynamic-layers/virtualization/recipes-azure-iot/iotedge/aziot-edged/aziot-edged.inc

```diff
@@ -17,6 +22,8 @@ export SOCKET_DIR = "/run/aziot"
 
 inherit aziot features_check omnect_rust_aziot-identityd_deps systemd
 
+RUSTFLAGS+="-D warnings"
```

Comment:
do we want to handle that more globally?

Reply:
as discussed, we will do it in another PR